build.gradle.kts

```diff
@@ -89,7 +90,7 @@ allprojects {
         withType<KotlinCompile> {
             kotlinOptions {
                 jvmTarget = javaVersion.toString()
-                languageVersion = rootProject.libs.plugins.kotlin.get().version.strictVersion
+                //languageVersion = rootProject.libs.plugins.kotlin.get().version.strictVersion
```

Comment:
Why is this commented out? Should have a TODO to add it back probably

Reply:
Added it back. Required a little extra parsing of the string, presumable due to changes in how gradle catalogs are handled. All works as normal though